AnswerGeneratorMicroservice/helper/model downloader.py

```diff
@@ -0,0 +1,27 @@
+from transformers import pipeline
+from transformers import AutoModelForSeq2SeqLM, AutoTokenizer
+from transformers import BitsAndBytesConfig
+import torch
+
+nf4_config = BitsAndBytesConfig(
```

Comment:
move this into configuration and let the configuration parameters be configurable

Reply:
Will do it when i will use  the LLM locally ,for no i will delete that trash files of LLM